### ISSUE TITLE
Move to billboard.js?

### ISSUE THREAD

**Issue by lbayerl** [GraFiddle/angular-chart#127]
As C3.js unfortunately isn't really maintained anymore. 

There is a promising fork existing now with [billboard.js](https://naver.github.io/billboard.js/) which 

1. more or less still directly supports c3.js options
2. has updated dependencies (e.g. d3 v4)
3. lacks a bit angular 1.x support (only inital light wrappers are existing like [billboard-ng(]https://github.com/fintechstudios/billboard-ng) )

I thought it could be a good idea to update angular-chart using billboard.js as a backbone.

What do you think? Are you open to any PRs?

**Comment by maxklenk** [comment]
Yes, billboard.js seam to get good support of the community. I would be very happy the receive a pull request which integrates billboard.js. I have not yet checked what has to be changed to integrate it.
